langchain/document_loaders/youtube.py

```diff
@@ -146,13 +146,15 @@ def __init__(
         self,
         video_id: str,
         add_video_info: bool = False,
-        language: str = "en",
+        language: List[str] = ["en"],
+        translation: str = "en",
         continue_on_failure: bool = False,
     ):
         """Initialize with YouTube video ID."""
         self.video_id = video_id
         self.add_video_info = add_video_info
         self.language = language
+        self.translation = translation
```

Comment:
```self.translation = translation if isinstance(translation, Sequence) else [translation]```

Reply:
hey @eyurtsev could you give more context about it.

In the [youtube transcript package](https://pypi.org/project/youtube-transcript-api/) it should string, not list
`translated_transcript = transcript.translate('de')`